artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -1130,6 +1136,10 @@ public void addHead(final MessageReference ref, boolean scheduling) {
    /* Called when a message is cancelled back into the queue */
    @Override
    public void addSorted(final MessageReference ref, boolean scheduling) {
+
+      if (RefCountMessage.isTraceEnabled()) {
+         RefCountMessage.deferredDebug(ref.getMessage(), "returning to queue {}", this.getAddress());
+      }
```

Comment:
The newline should be between the two entirely separate if's rather than before them.